core/main.go

```diff
@@ -56,15 +62,10 @@ func initDependencies() (*dependencies, error) {
 func deployCoreServices(cfg *config.Config, api *api.API) error {
 	for _, service := range cfg.Services() {
 		logrus.Infof("Deploying service %q from %q", service.Key, service.URL)
-		s, valid, err := api.DeployServiceFromURL(service.URL, service.Env)
-		if valid != nil {
-			return valid
-		}
+		s, err := api.DeployServiceFromURL(service.URL, service.Env, nil)
 		if err != nil {
 			return err
 		}
-		service.Sid = s.Sid
```

Comment:
These 2 lines of code are important in order to have the informations in the config especially when we get informations with the `Info` api